source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -1517,6 +1545,49 @@ let validate ({ Heap.sources; sinks; transforms; features; _ } as configuration)
     |> Result.combine_errors_unit
     |> Result.map_error ~f:List.concat
   in
+  let compare_locations previous_location location =
+    match previous_location, location with
+    | Some previous_location, Some location ->
+        JsonAst.LocationWithPath.equal previous_location location
+    | None, None -> true
+    | _ -> false
+  in
```

Comment:
Use `Option.equal JsonAst.LocationWithPath.equal`

Reply:
Thanks for that!